src/Utility/AddCommentTrait.php

```diff
@@ -23,8 +23,8 @@ protected function getClosestStatementNode(Node $node): ?Node {
         if ($node instanceof Node\Stmt) {
             $statement_node = $node;
         }
-        elseif ($node->hasAttribute('parentNode')) {
-            $parent_node = $node->getAttribute('parentNode');
+        elseif ($node->hasAttribute('parent')) {
```

Comment:
Btw, it's more future proof to use `AttributeKey::PARENT` constant

Reply:
:O yes, I will work on that fix.